global_helpers/panther_greynoise_helpers.py

```diff
@@ -84,15 +84,52 @@ def vpn_service(self, match_field) -> str:
         return deep_get(self.noise, match_field, "vpn_service")
 
 
-class GreyNoiseRIOT:
+class GreyNoiseRIOTBasic:
     def __init__(self, event):
         self.riot = deep_get(event, "p_enrichment", "greynoise_riot_basic")
 
-    def ip_range(self, match_field) -> str:
-        return deep_get(self.riot, match_field, "ip_cidr")
+    def ip_address(self, match_field) -> str:
+        return deep_get(self.riot, match_field, "provider", "ip")
+
+    def classification(self, match_field) -> str:
```

Comment:
There isn't a classification on the RIOT set.  Basically "RIOT" *is* the classification.

Reply:
Not sure where I picked that up, must have been looking at the wrong page of the API docs